frontend/components/WebsiteHeader.vue

```diff
@@ -1,21 +1,14 @@
 <template>
-  <header
-    ref="header"
-    class="sticky top-0 z-10 duration-500 header drop-shadow-md"
-    :class="{
-      'bg-light-header dark:bg-dark-header': headerOpacity == 1,
-      'bg-light-header/80 dark:bg-dark-header/80': headerOpacity == 0.8,
-      'invisible opacity-0': headerOpacity == 0,
-    }"
-  >
+  <header ref="header" class="sticky top-0 z-10 duration-500 header drop-shadow-md relative" :class="{
+    'bg-light-header dark:bg-dark-header': headerOpacity == 1,
+    'bg-light-header/80 dark:bg-dark-header/80': headerOpacity == 0.8,
+    'invisible opacity-0': headerOpacity == 0,
+  }">
     <!-- Note: mobile header -->
-    <div class="flex py-3 md:hidden">
-      <div class="z-0 flex justify-center mx-auto">
-        <div class="relative z-0 w-24 h-6">
-          <LogoActivist
-            :is-mobile="true"
-            class="absolute inset-0 flex items-center justify-center z-1 overflow-clip"
-          />
+    <div class="flex md:hidden py-3 px-4">
```

Comment:
Another minor thing I'm wondering about is why for the website header on mobile the focus for it is now the entire container rather than just `activist` 🤔

![Screenshot_2023-08-12 01 12 58](https://github.com/activist-org/activist/assets/24387426/69171a20-9350-4248-a23e-60f3248c627c)

Seems to be coming from here. Generally any edit that gets the focus back properly would be great 😊